README.md

```diff
@@ -31,7 +31,18 @@
 
 ## Exposing Elixir functions to Lua
 
-`Lua` provides the `deflua` macro for exposing Elixir functions to Lua
+The simplest way to expose an Elixir function to in Lua is using the `Lua.set!/3` function
```

Comment:
```suggestion
The simplest way to expose an Elixir function to Lua is using the `Lua.set!/3` function
```

Reply:
Also make a note that functions exposed this way must return lists, while functions using deflua are automatically wrapped.

Another consideration would be to auto wrap functions passed to set!